src/index.css

```diff
@@ -995,12 +998,72 @@ hr.horizontalLine.blue {
 .order-details-btn-link .p-button:enabled:hover {
   @apply border-none bg-inherit;
 }
-.quick-link-arrow img{
+.quick-link-arrow img {
   @apply h-7p w-7p;
 }
+.report-tab.p-tabview {
+  width: 335px;
+}
+.p-datatable .p-datatable-tfoot > tr > td {
+  @apply !bg-[#22262C] text-[#FFFFFF] border-0 border-[#292E36] p-3;
```

Comment:
Use the existing tailwind class (incase any) or extend it in the tailwind config.